io.elementary.videos.yml

```
@@ -17,9 +17,18 @@ finish-args:
   - '--talk-name=org.gnome.SettingsDaemon'
 
   - '--metadata=X-DConf=migrate-path=/io/elementary/videos/'
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '20.08'
```

```suggestion
    version: '21.08'
```

Is there a reason why you refer to the outdated version here?

---

The 21.08 version doesn't work with the 6 runtime, once videos is updated to use the 6.1 runtime this should be updated to 21.08

---

Thanks for clarification 🙂